spec/sidekiq/central_mail/submit_form4142_job_spec.rb

```diff
@@ -130,6 +130,11 @@
       end
 
       context 'with a client error' do
+        # Disable schema validation to test CentralMail error handling with invalid data
+        before do
+          Flipper.disable(:form4142_validate_schema)
```

Comment:
Don't enable/disable feature flags directly in the spec. I think this is what's causing the [test errors](https://depo-platform-documentation.scrollhelp.site/developer-docs/handling-flaky-unit-tests). Instead, mock the call, as we are doing in the other spec file and as described in the [Platform developer docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/handling-flaky-unit-tests).